ansible/roles/cyhy_mailer/vars/main.yml

```diff
@@ -5,3 +5,5 @@ reporter_user: "{{ lookup('aws_ssm', '/cyhy/mongo/users/reporter/user') }}"
 reporter_pw: "{{ lookup('aws_ssm', '/cyhy/mongo/users/reporter/password') }}"
 # reporter mongo database
 reporter_db: "{{ lookup('aws_ssm', '/cyhy/mongo/users/reporter/database') }}"
+# CSA region-to-email YAML mapping
+csa_email_yaml: "{{ lookup('aws_ssm', '/cyhy/csa_email_yaml') }}"
```

Comment:
While we are in here, we may as well alphabetize by variable name.

Reply:
Please see commit 2da8dff.